src/coreComponents/finiteVolume/SurfaceElementStencil.hpp

```diff
@@ -427,7 +427,7 @@ SurfaceElementStencilWrapper::
                   real64 (& dWeight_dVar1 )[maxNumConnections][2],
                   real64 (& dWeight_dVar2 )[maxNumConnections][2][3] ) const
 {
-  real64 sumOfTrans = 0.0;
+  real64 sumOfTrans = 1e-15;
```

Comment:
why do you need this change? can it actually be 0?

Reply:
if the aperture = 0 is the initial condition, the sumOfTrans will be zero and cause an invalid operation in division. So I just give it a small enough number to avoid division by zero.

Reply:
There are few other places with similar treatment, but I am not 100% sure what is the right way to deal with those division by zero.